README.md

```diff
@@ -16,6 +16,11 @@ Table of Contents
   - [Linux Specific Runtime](runtime-linux.md)
 - [Implementations](implementations.md)
 
+In the specifications in the above table of contents, the keywords "MUST",
+"MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT",
+"RECOMMENDED", "MAY", and "OPTIONAL" are to be interpreted as described in
+[RFC 2119](http://www.ietf.org/rfc/rfc2119.txt) (Bradner, S., "Key words for use in RFCs to Indicate Requirement Levels", BCP 14, RFC 2119, March 1997.)
```

Comment:
I prefer the HTML RFC renderings (e.g. [RFC 2199][1]), because they have anchors for linking, links to errata, obsoleting RFCs, etc.  They also have a link to the text and PDF renderings, so folks who would rather not view the HTML are a click away from their preferred format.

[1]: http://tools.ietf.org/html/rfc2119